sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -167,6 +167,7 @@ export interface DirectoryForceCloseHandlesHeaders {
     errorCode?: string;
     marker?: string;
     numberOfHandlesClosed?: number;
+    numberOfHandlesFailedToClosed?: number;
```

Comment:
>numberOfHandlesFailedToClosed [](start = 4, length = 29)

`numberOfHandlesFailedToClose` sounds better

Reply:
I think it comes directly from the generated code.
They have fixed this naming in the swagger, so a re-generate and re-build will fix this.